notebooks/240-dolly-2-instruction-following/240-dolly-2-instruction-following.ipynb

```diff
@@ -38,16 +38,17 @@
    "metadata": {},
```

Comment:
Introduction part should also be updated ("The tutorial consists of the following steps:")

Reply:
Done